pilot/pkg/model/service.go

```diff
@@ -609,18 +610,45 @@ func ParseServiceKey(s string) (hostname Hostname, ports PortList, labels Labels
 	return
 }
 
+// defaultSubsetName - Use this name for subset is not defined.
+const defaultSubsetName = "_default_"
```

Comment:
How about "_" or "_default" ( without the trailing _)

Again - I'm looking at common naming patterns for DNS (SRV records or DNSSD)